src/garage/tf/policies/gaussian_mlp_policy2.py

```diff
@@ -0,0 +1,271 @@
+        self._init_std = init_std
+        self._min_std = min_std
+        self._max_std = max_std
+        self._std_hidden_sizes = std_hidden_sizes
+        self._std_hidden_nonlinearity = std_hidden_nonlinearity
+        self._std_output_nonlinearity = std_output_nonlinearity
+        self._std_parameterization = std_parameterization
+        self._layer_normalization = layer_normalization
+
+        self.model = GaussianMLPModel2(
+            output_dim=self.action_dim,
+            hidden_sizes=hidden_sizes,
+            hidden_nonlinearity=hidden_nonlinearity,
+            hidden_w_init=hidden_w_init,
+            hidden_b_init=hidden_b_init,
+            output_nonlinearity=output_nonlinearity,
+            output_w_init=output_w_init,
+            output_b_init=output_b_init,
+            learn_std=learn_std,
+            adaptive_std=adaptive_std,
+            std_share_network=std_share_network,
+            init_std=init_std,
+            min_std=min_std,
+            max_std=max_std,
+            std_hidden_sizes=std_hidden_sizes,
+            std_hidden_nonlinearity=std_hidden_nonlinearity,
+            std_output_nonlinearity=std_output_nonlinearity,
+            std_parameterization=std_parameterization,
+            layer_normalization=layer_normalization,
+            name='GaussianMLPModel')
+
+    # pylint: disable=attribute-defined-outside-init
+    def build(self, state_input, name=None):
+        """Build model.
+
+        Args:
+          state_input (tf.Tensor) : State input.
+          name (str): Name of the model, which is also the name scope.
+
+        """
+        with tf.compat.v1.variable_scope(self.name) as vs:
+            self._variable_scope = vs
+            self.model.build(state_input, name=name)
+
+            self._f_dist = tf.compat.v1.get_default_session().make_callable(
+                [
+                    self.model.networks['default'].mean,
+                    self.model.networks['default'].log_std
+                ],
+                feed_list=[self.model.networks['default'].input])
+
+    @property
+    def vectorized(self):
+        """Vectorized or not.
+
+        Returns:
+            Bool: True if primitive supports vectorized operations.
+
+        """
+        return True
+
+    def get_action(self, observation):
+        """Get single action from this policy for the input observation.
+
+        Args:
+            observation (numpy.ndarray): Observation from environment.
+
+        Returns:
+            numpy.ndarray: Actions
+            dict: Predicted action and agent information.
+
+        Note:
+            It returns an action and a dict, with keys
+            - mean (numpy.ndarray): Distribution parameter.
+            - log_std (numpy.ndarray): Distribution parameter.
+
+        """
+        mean, log_std = self._f_dist([observation])
+        rnd = np.random.normal(size=mean.shape)
+        sample = rnd * np.exp(log_std) + mean
```

Comment:
rather than doing this, you can just call `dist.sample()` in your feed

Reply:
fixed